packages/surveys/src/utils/results-availability/has-user-responded.ts

```diff
@@ -0,0 +1,25 @@
+import { queryFeatures, IQueryResponse } from "@esri/arcgis-rest-feature-layer";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IStakeholderItem } from "../../types";
+
+/**
+ * check is provided user has any survey submissions
```

Comment:
I think some more explanation of this method purpose would be helpful. It is likely useful for many future cases, but from the design & the tests (yay) it seems this is something like `Example: used to determine if resultsAvailability is set to 'after'`

Reply:
Yes, there will be more values of `resultsAvailability` to come